packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -213,23 +223,48 @@ export class ForceApexLogList {
   }
 }
 
-export class ApexLibraryGetLogsExecutor extends LibraryCommandletExecutor<{ id: string }> {
+export class ApexLibraryGetLogsExecutor extends LibraryCommandletExecutor<{ id: string; }> {
   protected executionName: string = nls.localize('apex_log_get_text');
   protected logName: string = 'force_apex_log_get_library';
 
-  protected async run(response: ContinueResponse<{ id: string }>): Promise<boolean> {
+  protected async run(
+    response: ContinueResponse<{ id: string }>
+  ): Promise<LibraryExecution> {
     const connection = await workspaceContext.getConnection();
+    // @ts-ignore
     const logService = new LogService(connection);
     const { id: logId } = response.data;
-    const outputDir = path.join(getRootWorkspacePath(), SFDX_FOLDER, 'tools', 'debug', 'logs');
+    const outputDir = path.join(
+      getRootWorkspacePath(),
+      SFDX_FOLDER,
+      'tools',
+      'debug',
+      'logs'
+    );
 
     await logService.getLogs({ logId, outputDir });
 
     const logPath = path.join(outputDir, `${logId}.log`);
     const document = await vscode.workspace.openTextDocument(logPath);
     vscode.window.showTextDocument(document);
 
-    return true;
+    return { success: true };
+  }
+}
+
+export class ApexLibraryLogListExecutor extends LibraryCommandletExecutor<{}, LogRecord[]> {
+  protected executionName: string = nls.localize('apex_log_list_text');
+  protected logName: string = 'force_apex_log_list_library';
+
+  protected async run(
+    response: ContinueResponse<{}>
+  ): Promise<LibraryExecution<LogRecord[]>> {
+    const connection = await workspaceContext.getConnection();
+    // @ts-ignore
+    const logService = new LogService(connection);
+    const logs = await logService.getLogRecords();
```

Comment:
This is part of the changes that will be released this week on the apex library. Which means we also need to update the version of it on the package.json and hold on merging the PR until that release happens